ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -34,7 +34,7 @@ tests:
       TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
       TEST_RELEASE_TYPE: Pre-GA
       TEST_SCENARIOS: sig-kata.*Kata Author
-      TEST_TIMEOUT: "75"
+      TEST_TIMEOUT: "90"
```

Comment:
Would you please describe in a commit message why such change is required so we can understand and ack that?